nav2_bt_navigator/behavior_trees/navigate_w_replanning_only_if_path_becomes_invalid.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+  This Behavior Tree replans the global path only if the path becomes invalid
+-->
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <PipelineSequence name="NavigateWithReplanning">
+      <Fallback> 
+        <ReactiveSequence>
+          <Inverter>
+            <GlobalUpdatedGoal/>
+          </Inverter>
+          <IsPathValid path="{path}"/>
```

Comment:
Question - Should this be throttled? How often is this running (at the tick rate of 100hz)?

Reply:
Done